src/vs/workbench/services/editor/common/editorGroupsService.ts

```diff
@@ -767,6 +767,36 @@ export interface IEditorGroup {
 	 */
 	isActive(editor: EditorInput | IUntypedEditorInput): boolean;
 
+	/**
+	 * Selects the editor in the group. If the editor is not specified, selects the active editor.
+	 */
+	selectEditor(editor?: EditorInput): void;
```

Comment:
I think these new methods can go on the internal interface because I would not assume they are used outside? Or are we planning to add commands?